hedera-node/hedera-mono-service/src/test/java/com/hedera/services/ledger/interceptors/LinkAwareTokenRelsCommitInterceptorTest.java

```diff
@@ -81,6 +84,7 @@ void noChangesAreNoop() {
     void tracksNothingIfOpIsNotAutoAssociating() {
         given(accessor.getFunction()).willReturn(CryptoUpdate);
         given(txnCtx.accessor()).willReturn(accessor);
+        given(tokenRelSupplier.get()).willReturn(new OnDiskTokenRel());
```

Comment:
I'm guessing that it doesn't matter whether it's an `OnDiskTokenRel` or an in-memory `MerkleTokenRel` for this and the other tests in this class?

Reply:
True.